src/app/users/active-users.ts

```diff
@@ -0,0 +1,42 @@
+import { MS_PER_30_DAYs, USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD } from "@config/app"
+import { baseLogger } from "@services/logger"
+import { User } from "@services/mongoose/schema"
```

Comment:
App should not depend on mongoose that is what the repository is for - an adapter between domain and persistence.